tests/src/python/test_qgsrendererrasterpropertieswidget.py

```diff
@@ -0,0 +1,69 @@
+import pathlib
+from qgis.testing import start_app, unittest, TestCase
+from qgis.testing.mocked import get_iface
+from qgis.core import QgsRasterLayer, QgsProject, QgsMultiBandColorRenderer, QgsRasterRenderer, QgsSingleBandGrayRenderer
+from qgis.gui import QgsRendererRasterPropertiesWidget, QgsMapCanvas, QgsMultiBandColorRendererWidget, QgsRasterRendererWidget
+
+
+class QgsRendererRasterPropertiesTestCases(TestCase):
```

Comment:
These new tests need to be added to the cmake file in the same folder, or they won't be run as part of the test suite

Reply:
I added these lines, but travis-ci fails because of another test not affected by this PR:
````
The following tests FAILED:
	327 - PyQgsExpressionBuilderWidget (Failed)
File "/tmp/tmpckpqqseh/highlight_lines.png" not found.
File "/tmp/tmp247kzd7r/highlight_polygons.png" not found.
	Add file: /root/QGIS/.ci/travis/travis.ctest
	Add file: /root/QGIS/build/CMakeCache.txt
````